src/__tests__/rockets.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../redux/configureStore';
+import Rockets from '../pages/Rockets';
+
+describe('Test for Rockets components', () => {
+  it('Check if the Rockets section are available', () => {
+    const rockets = renderer.create(
+      <Provider store={store}>
+        <Rockets />
+      </Provider>,
+    );
+    expect(rockets).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Great job adding test for rockets, However I suggest to add test for all other your components and functions such as Header,Missions, Profile, ...

![image](https://user-images.githubusercontent.com/45231276/202426285-0d00dc9c-8f64-40ac-ab6f-46af575358e7.png)

Reply:
Hi @Bondok6,
I am a bit confused by this as I have tests for the Header, Missions, and Profile?

Reply:
Thank you, I See i forgot to push my last commit.